app/views/products/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>All Costumes</h1>
+	<% @products.each do |product| %>
+<div class="all-products">
+	<div class="favoritepic">
+		<% unless product.photo_url.nil? %>
+		<%= link_to (image_tag product.photo_url), product_path(product.id) %>
+		<% end %><br>
+	</div>
+	<%= link_to product.name, product_path(product.id) %><br>
+	$<%= number_with_precision product.price, precision: 2 %><br>
+
+		<%= form_tag '/orderitems' do  %>
+			<%= hidden_field_tag :product_id, product.id %> <br>
+			<%= select_tag :quantity, options_for_select(1..product.quantity) %><br>
+  <% if product.quantity == 0 %>
+  	<p class=retired-empty>OUT OF STOCK!</p>
+  <% else %>
+    <p class="order-quantity"><%= product.quantity %> Available!</p>
+
+	<%= submit_tag "Add to Cart", class: 'btn btn-primary' %><br>
+	<% end %>
+<hr>
+		<% end %>
+</div>
+	<% end %>
```

Comment:
Woah, this code is jumping all over the page! Sorry to keep harping on this but proper indentation makes unfamiliar code so much easier to read, it's best to get in the habit as soon as you can.